packages/swap/src/routes/quote.ts

```diff
@@ -38,6 +38,117 @@ const handleQuotingError = (res: express.Response, err: unknown, info: Additiona
   res.status(500).json({ message });
 };
 
+export const getDcaQuoteParams = async (asset: InternalAsset, amount: bigint) => {
+  const usdChunkSize = env.DCA_USD_CHUNK_SIZE;
```

Comment:
lets create a map of asset to `usdChunksize` so we can change it easily on the go per asset in the future.